block-producer/src/store/mod.rs

```diff
@@ -45,6 +46,7 @@ pub trait ApplyBlock: Send + Sync + 'static {
 }
 
 /// Information needed from the store to verify a transaction.
+#[derive(Debug)]
 pub struct TxInputs {
     /// The account hash in the store corresponding to tx's account ID
     pub account_hash: Option<Digest>,
```

Comment:
I would probably manually implement `Display` on this to make sure we get good output in the logs.

Reply:
@bobbinth I've implemented `Debug` for protobuf's `Digest` and it looks nice now. The only issue is with `RpoDigest`, will have to handle them manually.

Reply:
I believe `RpoDigest` has `Display` implemented on it which prints out the value in hex.

Reply:
@bobbinth 
> I believe `RpoDigest` has `Display` implemented on it which prints out the value in hex.

Yes, but in a case when digest is a field in structure, I can't specify which trait to use when logging it.